lib/pkcs11h-threading.c

```diff
@@ -388,8 +388,7 @@ _pkcs11h_threading_condInit (
 	if (
 		(
 			pthread_mutex_init (&cond->mut, NULL) ||
-			pthread_cond_init (&cond->cond, NULL) ||
-			pthread_mutex_lock (&cond->mut)
+			pthread_cond_init (&cond->cond, NULL)
```

Comment:
we need to initialize the mutex somewhere, no?

Reply:
It is initialized right here, on line 390. I only removed its locking right after initialization, because this happens in the main thread.